src/federated-search/pages/index.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { InstantSearch, Index, Configure } from 'react-instantsearch-hooks-web'
+import { instantMeiliSearch } from '@meilisearch/instant-meilisearch'
+import Header from '../components/Header'
+import MovieResults from '../components/Results/MovieResults'
+import ActorResults from '../components/Results/ActorResults'
+
+const searchClient = instantMeiliSearch(
+  process.env.NEXT_PUBLIC_MEILISEARCH_HOST,
+  process.env.NEXT_PUBLIC_MEILISEARCH_SEARCH_API_KEY
+)
+
+const App = () => (
+  <InstantSearch indexName="movies" searchClient={searchClient}>
+    <Header />
+    <div class="title">
+      <h1>Federated search</h1>
+      <p>This demo showcases federated search in action.</p>
```

Comment:
I would just update the wording here to explain what federated search does, ie. searching across multiple indexes, so that users that are less knowledgeable still get it